.gitignore

```diff
@@ -27,3 +27,4 @@ deps/lua/src/liblua.a
 .make-*
 .prerequisites
 *.dSYM
+TAGS
```

Comment:
Why did you add this?

Reply:
looks like this was the original commit, and the error handling cleanup came later. ¯\_(ツ)_/¯
since it mixes 2 different things, opener isn't responsive, and this change causes a conflict. i'm dismissing it and merging the other change.